app/assets/javascripts/userdash.js

```diff
@@ -55,7 +60,11 @@ require(["lib/models", "lib/tools", "knockout", "sammy"], function (models, tool
 
             //conference data
             function onConferenceData(confObj) {
-                console.log("+ onConferenceData")
+                console.log("+ onConferenceData");
+                if((typeof confObj === 'string' || confObj instanceof String) && confObj.length == 0){
+                    self.setInfo("info","You have no own abstracts created yet. Create one via accessing a conference page.");
```

Comment:
Please remove the part `Create one via accessing a conference page`, since there might be no conferences where the submission is open.